### PR TITLE
feat: mevbot integrate

### PR DESCRIPTION
**Pull request by namgold**

### CODE REVIEW

File: src/components/SwapForm/SlippageSettingGroup.tsx [hunk 1]
```diff
@@ -35,34 +34,44 @@ export default function SlippageSettingGroup({
...
-  const currencyIn = useInputCurrency()
-  const currencyOut = useOutputCurrency()
-  const { typedValue } = useSwapState()
   const { chainId } = useActiveWeb3React()
+  const [showMevModal, setShowMevModal] = useState(false)
+  const addMevProtectionHandler = useCallback(() => {
+    setShowMevModal(true)
+  }, [])
+
+  const rightButton =
```
Comment (viet-nv) [inline src/components/SwapForm/SlippageSettingGroup.tsx:43]: did we permanent remove the priceAlert button? 
cc @nguyenhoaidanh

File: src/components/SwapForm/AddMEVProtectionModal.tsx [hunk 2]
```diff
@@ -0,0 +1,149 @@
...
+        <Row gap="16px" flexDirection={upToExtraSmall ? 'column' : 'row'}>
+          {rpcOptions.map(({ name, logo }) => (
+            <RPCOption selected={selectedRpc === name} key={name} onClick={() => setSelectedRpc(name)}>
+              <img src={logo} />
+              {name}
+            </RPCOption>
+          ))}
+        </Row>
+        <Row gap="16px" flexDirection={upToExtraSmall ? 'column' : 'row'}>
+          <ButtonOutlined onClick={onClose}>No, go back</ButtonOutlined>
```
Comment (viet-nv) [inline src/components/SwapForm/AddMEVProtectionModal.tsx:141]: Trans

File: src/components/SwapForm/AddMEVProtectionModal.tsx [hunk 3]
```diff
@@ -0,0 +1,149 @@
...
+          <ButtonPrimary onClick={onAdd} disabled={!selectedOption}>
+            Yes
```
Comment (viet-nv) [inline src/components/SwapForm/AddMEVProtectionModal.tsx:143]: Trans

File: src/components/SwapForm/index.tsx [hunk 4]
```diff
@@ -186,7 +208,28 @@ const SwapForm: React.FC<SwapFormProps> = props => {
                 )}
               </Flex>
 
-              <ReverseTokenSelectionButton onClick={() => currencyIn && handleChangeCurrencyOut(currencyIn)} />
+              <Flex sx={{ gap: '12px' }}>
+                <Flex alignItems="flex-start" fontSize={12} color={theme.subText} justifyContent="space-between">
+                  <PriceAlertButton
```
Comment (viet-nv) [inline src/components/SwapForm/index.tsx:213]: Remove price alert on linea testnet.

Comment (viet-nv) [reply src/components/SwapForm/index.tsx:213]: btw, keep it is ok, we will remove this network soon

**Review by nguyenhoaidanh — APPROVED** [review]

**Pull request merged**